sys-apps/systemd/systemd-9999.ebuild

```diff
@@ -24,11 +24,11 @@ inherit autotools bash-completion-r1 linux-info multilib \
 	multilib-minimal pam systemd toolchain-funcs udev user
 
 DESCRIPTION="System and service manager for Linux"
-HOMEPAGE="http://www.freedesktop.org/wiki/Software/systemd"
+HOMEPAGE="https://www.freedesktop.org/wiki/Software/systemd"
 
 LICENSE="GPL-2 LGPL-2.1 MIT public-domain"
 SLOT="0/2"
-IUSE="acl apparmor audit cryptsetup curl elfutils gcrypt gnuefi http
+IUSE="acl apparmor audit cryptsetup curl elfutils +gcrypt gnuefi http
```

Comment:
This was just to bring us closer to upstream's build, not for any specific issues.  We're installing libgcrypt, and enabling it in systemd just seems to provide better security for the  journal and dnssec.  It can be dropped if we don't want it.